engine/installation/linux/oracle.md

```diff
@@ -106,29 +137,21 @@ Docker from the repository.
     > `yum install` or `yum upgrade` command will always install the highest
     > available version, which will almost certainly be an unstable one.
 
-3.  On production systems, you should install a specific version of Docker
+4.  On production systems, you should install a specific version of Docker
```

Comment:
There is no explanation of why you should install a specific version of Docker, nor how to decide which version to install. In the absence of this decision-making guidance, the best advice should be to install the latest stable version, which is what would happen automatically if the `testing` repository is not enabled.